### PR TITLE
new changes

### PR DESCRIPTION
**Pull request by pravatvmware1**
hello new changes.
@pravatvmware can you plz have a look.

### CODE REVIEW

**Comment by pravatvmware** [comment]
new command added.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # world1
+new line added
 learn to create repo with pull request
 
-New line added by user pravatvmware1
 
 Hello
 Second line added.
+
+Love you coding style. All good.
```
Comment (pravatvmware) [inline README.md:9]: Why this changes now?

File: README.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
 # world1
+new line added
```
Comment (pravatvmware) [inline README.md:2]: don't put here.

**Review by pravatvmware — COMMENTED** [review]
can incorporate this new changes.

File: README.md (hunk 2 above)
Comment (pravatvmware1) [reply README.md:2]: ok

File: README.md (hunk 1 above)
Comment (pravatvmware1) [reply README.md:9]: good

**Review by pravatvmware — COMMENTED** [review]
ok
